clang/test/CodeGenSYCL/address-space-swap.cpp

```diff
@@ -1,34 +0,0 @@
-// RUN: %clang -fsycl-device-only -S -Xclang -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
```

Comment:
Did you mean to delete this test?

Reply:
Yes, I meant it. It basically tests the conversion of address spaces (from default to generic) that happens when `std::swap` is called. However such conversion happens always when any function is called with default address space parameters and these cases are covered by `address-space-parameter-conversions.cpp` test, so I don't see the value in the test which tests already covered functionality but brings dependency on standard header and clang driver.

Reply:
However, I have to fix Windows fails. I didn't mean them :)